packages/cms/src/components/Viz/index.jsx

```diff
@@ -19,6 +19,8 @@ class Viz extends Component {
 
     const variables = this.props.variables || this.context.variables;
     const locale = this.props.locale || this.context.locale;
+    const id = this.props.id || "No ID Provided to Viz Component";
```

Comment:
not exactly what I meant... should we not just use `config.id`?

Reply:
Sure we can move it into the config itself, but either way, the user would be responsible with loading up the config with this parameter.  It's easy for me to do this in the VisualizationCards of the CMS, but when people are making vizes on the front end, they will need to get the id from the original json payload, and inject it into the config object when they instantiate a `Viz`, otherwise this will come up null (Remember, the `logic` itself does not have an id in it).

Either way, having it as part of the config rather than a prop is cleaner - making that change.